packages/skeleton-svelte/src/lib/components/Progress/Progress.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+	import type { ProgressProps } from './types.js';
+
+	let {
+		value,
+		min = 0,
+		max = 100,
+		// Root
+		base = 'overflow-x-hidden bg-surface-200-800',
+		height = 'h-2',
+		rounded = 'rounded',
+		classes = '',
+		// Indicator
+		indicatorBase = 'bg-primary-500 h-full',
+		indicatorRounded = 'rounded',
+		indicatorTransition = 'transition-[width]',
+		indicatorIndeterminateAnimation = 'indeterminate',
+		indicatorClasses = ''
+	}: ProgressProps = $props();
+
+	const indeterminate = $derived(value === undefined);
+	const fillPercentage = $derived(indeterminate ? 50 : ((value! - min) / (max - min)) * 100);
+	const width = $derived(`${fillPercentage}%`);
+</script>
+
+<!-- @component Track -->
+<div class="{base} {height} {rounded} {classes}">
+	<!-- @component Indicator -->
+	<div
+		class="{indicatorBase} {indicatorRounded} {indicatorTransition} {indeterminate
+			? indicatorIndeterminateAnimation
+			: ''} {indicatorClasses}"
```

Comment:
since the ternary operator here has only 1 option, I would prefer to use this instead:
```suggestion
		class="{indicatorBase} {indicatorRounded} {indicatorTransition} {indeterminate && indicatorIndeterminateAnimation} {indicatorClasses}"
```
This is still readable and cleaner IMHO.

Reply:
I had this before, but this would create a `false` class, not what you want

Reply:
> I had this before, but this would create a `false` class, not what you want

Are you sure? I tested it yesterday and it worked as expected!

Reply:
It does work but when `indeterminate` is `false` it will literally add a `false` string to the classes.

Reply:
Done. Moved to an `rx` variable that's either the animation or an empty string.